WordPress/Classes/Services/ReaderCardService.swift

```diff
@@ -27,25 +27,32 @@ class ReaderCardService {
             self.service.fetchCards(for: slugs,
                                success: { [weak self] cards in
 
-                                guard let syncContext = self?.syncContext else {
+                                guard let `self` = self else {
```

Comment:
can just be self 😬

Reply:
Done in a3899f4642ab0476001331fb02ff075193b70cbf